### DISCUSSION TITLE
top h drem

### DISCUSSION THREAD

**Discussion by serifonium** [serifonium/The-Expanse#2] [Show and tell]
![nick](https://user-images.githubusercontent.com/104041484/164337012-d9604e5a-5ad0-4598-9f2d-4af3363223f1.jpeg)

**Comment by nekoify** [comment]
when i was ten yrs old my goofy ahh uncle made me touch his

**Reply by serifonium** [reply]
<img width="607" alt="Screen Shot 2022-04-20 at 1 30 33 pm" src="https://user-images.githubusercontent.com/104041484/164337158-b9ebe245-53eb-4916-9d82-d42780e7b23e.png">

**Reply by Quantaviousdingleberry420** [reply]
balls
